code/processes/idb.q

```diff
@@ -0,0 +1,46 @@
+/-default parameters
+\d .idb
+
+reload:{[pt]
+    .lg.o[`reload;"reload command has been called remotely with partition: ", string pt];
+    .lg.o[`reload;"Doing nothing."];
+    .lg.o[`reload;"Finished reloading IDB"];
+ };
+
+/- loads the idb and the sym file. Called by the wdb process after every writedown.
+intradayreload:{[pt]
+    ptdir: ` sv  .idb.savedir,(`$string[pt]),`;
+    if[0=count key ptdir;:()];
+    system"l ", 1_string ptdir;
```

Comment:
Could we have an error trap / logging around this load command

Reply:
Sure, added now.